backend/src/schema/resolvers/user_management.js

```diff
@@ -5,7 +5,7 @@ import { neo4jgraphql } from 'neo4j-graphql-js'
 
 export default {
   Query: {
-    isLoggedIn: (parent, args, { driver, user }) => {
+    isLoggedIn: (_, args, { driver, user }) => {
```

Comment:
I would prefer `_parent` here, to keep clear where this parameter stands for. Just my opinion …

Reply:
I agree with you, I think it's not such a big deal cause there are other examples in the code base where someone looking for the naming would find it, but if we were to refactor all of them to have just an underscore, it would make it more difficult for people unfamiliar with the name convention to figure out what was going on